proto/attestation/reference_value.proto

```diff
@@ -53,6 +53,24 @@ message BinaryReferenceValue {
   }
 }
 
+message KernelDigests {
```

Comment:
Could you point to the tool to generate these digests, or some other place that explains their semantics?